WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutHeaderView.swift

```diff
@@ -0,0 +1,128 @@
+        let appIconCornerRadius: CGFloat
+    }
+
+    struct Fonts {
+        let appName: UIFont
+        let appVersion: UIFont
+    }
+
+    // MARK: - Defaults
+
+    public static let defaultSizing = Sizing(
+        appIconWidthAndHeight: CGFloat(80),
+        appIconCornerRadius: CGFloat(13))
+
+    public static let defaultSpacing = Spacing(
+        betweenAppIconAndAppNameLabel: CGFloat(16),
+        betweenAppNameLabelAndAppVersionLabel: CGFloat(4),
+        aboveAndBelowHeaderView: CGFloat(64))
+
+    // MARK: - View Customization
+
+    private let appInfo: AppInfo
+    private let spacing: Spacing
+    private let sizing: Sizing
+    private let fonts: Fonts
+
+    // MARK: - Initializers
+
+    init(appInfo: AppInfo,
+         sizing: Sizing = defaultSizing,
+         spacing: Spacing = defaultSpacing,
+         fonts: Fonts) {
+
+        self.appInfo = appInfo
+        self.sizing = sizing
+        self.spacing = spacing
+        self.fonts = fonts
+
+        super.init(frame: .zero)
+
+        setupSubviews()
+    }
+
+    override init(frame: CGRect) {
+        fatalError("Initializer not implemented!")
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("Initializer not implemented!")
+    }
+
+    // MARK: - Setting up the subviews
+
+    func setupSubviews() {
+        let stackView = UIStackView()
+        let iconView = UIImageView()
+        let appNameLabel = UILabel()
+        let appVersionLabel = UILabel()
+
+        clipsToBounds = true
+
+        appNameLabel.text = appInfo.name
+        appNameLabel.lineBreakMode = .byWordWrapping
+        appNameLabel.numberOfLines = 1
+        appNameLabel.font = fonts.appName
+
+        appVersionLabel.text = appInfo.version
+        appVersionLabel.lineBreakMode = .byWordWrapping
+        appVersionLabel.numberOfLines = 1
+        appVersionLabel.font = fonts.appVersion
+        appVersionLabel.textColor = UIColor.colorFromHex("8A898E")
+
+        iconView.image = appInfo.icon
+        iconView.layer.cornerRadius = sizing.appIconCornerRadius
+        iconView.layer.masksToBounds = true
+
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.addArrangedSubviews([
+            iconView,
+            appNameLabel,
+            appVersionLabel,
+        ])
+        stackView.setCustomSpacing(spacing.betweenAppIconAndAppNameLabel, after: iconView)
+        stackView.setCustomSpacing(spacing.betweenAppNameLabelAndAppVersionLabel, after: appNameLabel)
+
+        addSubview(stackView)
+
+        NSLayoutConstraint.activate([
+            iconView.centerXAnchor.constraint(equalTo: stackView.centerXAnchor),
+            iconView.heightAnchor.constraint(equalToConstant: sizing.appIconWidthAndHeight),
+            iconView.widthAnchor.constraint(equalToConstant: sizing.appIconWidthAndHeight),
+        ])
+
+        NSLayoutConstraint.activate([
```

Comment:
It might be neater to put all of these constraints in a single call to `activate`. You could always just add a blank line in between the groups of constraints to separate them.

Reply:
I actually did that because I wanted to split the logic in different methods, which IMHO makes things a bit cleaner.

I'll propose what I had in mind as an alternative.

Reply:
Nevermind, just pushed your requested change.